integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -696,24 +677,8 @@ public int rmdir(String path) {
    */
   @Override
   public int truncate(String path, long size) {
-    final AlluxioURI uri = mPathResolverCache.getUnchecked(path);
-    try {
-      if (!mFileSystem.exists(uri)) {
-        LOG.error("File {} does not exist", uri);
-        return -ErrorCodes.ENOENT();
-      }
-    } catch (IOException e) {
-      LOG.error("IOException encountered at path {}", path, e);
-      return -ErrorCodes.EIO();
-    } catch (AlluxioException e) {
-      LOG.error("AlluxioException encountered at path {}", path, e);
-      return -ErrorCodes.EFAULT();
-    } catch (Throwable e) {
-      LOG.error("Unexpected exception at path {}", path, e);
-      return -ErrorCodes.EFAULT();
-    }
-    LOG.error("File {} exists and cannot be overwritten. Please delete the file first", uri);
-    return -ErrorCodes.EEXIST();
+    LOG.error("Cannot change size of {}", path);
```

Comment:
Do you mind making this error more informative (ie. `truncate is not supported`)?

Reply:
Make sense, will do it.